tests/suites/os/tests/chrony/index.js

```diff
@@ -5,63 +5,43 @@ module.exports = {
 			title: 'Chronyd service',
 			run: async function(test) {
 				test.comment(`checking for chronyd service...`);
-				let result = '';
-				await this.context.get().utils.waitUntil(async () => {
-					result = await this.context
-						.get()
-						.worker.executeCommandInHostOS(
-							`systemctl is-active chronyd.service`,
-							this.context.get().link,
-						);
-					return result === 'active';
-				}, false);
-				result = await this.context
-					.get()
-					.worker.executeCommandInHostOS(
-						'systemctl status chronyd | grep running',
```

Comment:
@markcorbinuk This looks redundant to me, but I wanted to check with you first. I would think waiting for `systemctl is-active chronyd.service` would suffice to ensure the service is running, but maybe I'm missing some context.